data/json/professions.json

```
@@ -774,6 +774,40 @@
       "female": [ "panties", "bra", "stockings", "dress_shoes", "maid_dress", "maid_hat" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "captive",
+    "name": { "male": "Captive", "female": "Captive" },
+    "description": "You were folowing a road at night trying to get away from the horrors of the city when you heard a voice calling out in the dark.  As you stepped away to investigate you suddenly felt a seering pain in your head and blacked out.  You just woke up in this place... Are you even on earth anymore?",
+    "points": 0,
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "rescuer",
+    "name": { "male": "Rescuer", "female": "Rescuer" },
+    "description": "You were ready, you went in heads on determined to find and rescue your friends.  But now as you walk through those strange corridors the atmosphere grows heavy on you and you're not so sure anymore, you might be the one in need of a rescue now.",
```

```suggestion
    "description": "You were ready.  You went in determined to find and rescue your friends.  But now as you walk through those strange corridors, the atmosphere grows heavyand you're not so sure anymore. You might be the one in need of a rescue now.",
```
Reads a little better this way, but i'm not super clear what you meant by "you went in heads on determined" so it's possible that could be reworded properly.